docs/technical_reference/property_models/modified_ADM1.rst

```diff
@@ -0,0 +1,341 @@
+   "Uptake of hydrogen", ":math:`\rho_{11} = k_{m_{h2}} \frac{C_{S_{h2}}}{K_{S_{h2}}+C_{S_{h2}}} C_{X_{h2}} \cdot \frac{1}{1 + K_{S_{IN}}/C_{S_{IN}}/14} \cdot \frac{1}{1 + K_{S_{IP}}/C_{S_{IP}}/31} I_{pH,h2}`"
+   "Decay of X_su", ":math:`\rho_{12} = k_{dec, X_{su}} C_{X_{su}}`"
+   "Decay of X_aa", ":math:`\rho_{13} = k_{dec, X_{aa}} C_{X_{aa}}`"
+   "Decay of X_fa", ":math:`\rho_{14} = k_{dec, X_{fa}} C_{X_{fa}}`"
+   "Decay of X_c4", ":math:`\rho_{15} = k_{dec, X_{c4}} C_{X_{c4}}`"
+   "Decay of X_pro", ":math:`\rho_{16} = k_{dec, X_{pro}} C_{X_{pro}}`"
+   "Decay of X_ac", ":math:`\rho_{17} = k_{dec, X_{ac}} C_{X_{ac}}`"
+   "Decay of X_h2", ":math:`\rho_{18} = k_{dec, X_{h2}} C_{X_{h2}}`"
+   "Storage of S_va in X_PHA", ":math:`\rho_{19} = q_{PHA} \frac{C_{S_{va}}}{K_{A} + C_{S{va}}} \cdot \frac{C_{X_{PP}} / C_{X_{PAO}}}{K_{PP} + \frac{C_{X_{PP}}}{C_{X_{PAO}}}} C_{X_{PAO}} \frac{C_{S_{va}}}{C_{S_{va}} + C_{S_{bu}} + C_{S_{pro}} + C_{S_{ac}}}`"
+   "Storage of S_bu in X_PHA", ":math:`\rho_{20} = q_{PHA} \frac{C_{S_{bu}}}{K_{A} + C_{S{bu}}} \cdot \frac{C_{X_{PP}} / C_{X_{PAO}}}{K_{PP} + \frac{C_{X_{PP}}}{C_{X_{PAO}}}} C_{X_{PAO}} \frac{C_{S_{bu}}}{C_{S_{va}} + C_{S_{bu}} + C_{S_{pro}} + C_{S_{ac}}}`"
+   "Storage of S_pro in X_PHA", ":math:`\rho_{21} = q_{PHA} \frac{C_{S_{pro}}}{K_{A} + C_{S{pro}}} \cdot \frac{C_{X_{PP}} / C_{X_{PAO}}}{K_{PP} + \frac{C_{X_{PP}}}{C_{X_{PAO}}}} C_{X_{PAO}} \frac{C_{S_{pro}}}{C_{S_{va}} + C_{S_{bu}} + C_{S_{pro}} + C_{S_{ac}}}`"
+   "Storage of S_ac in X_PHA", ":math:`\rho_{22} = q_{PHA} \frac{C_{S_{ac}}}{K_{A} + C_{S{ac}}} \cdot \frac{C_{X_{PP}} / C_{X_{PAO}}}{K_{PP} + \frac{C_{X_{PP}}}{C_{X_{PAO}}}} C_{X_{PAO}} \frac{C_{S_{ac}}}{C_{S_{va}} + C_{S_{bu}} + C_{S_{pro}} + C_{S_{ac}}}`"
+   "Lysis of X_PAO", ":math:`\rho_{23} = b_{PAO} C_{X_{PAO}}`"
+   "Lysis of X_PP", ":math:`\rho_{24} = b_{PP} C_{X_{PP}}`"
+   "Lysis of X_PHA", ":math:`\rho_{25} = b_{PHA} C_{X_{PHA}}`"
+
+
+The rules for pH inhibition of amino-acid-utilizing microorganisms (:math:`I_{pH,aa}`), acetate-utilizing microorganisms (:math:`I_{pH,ac}`), and hydrogen-utilizing microorganisms (:math:`I_{pH,h2}`) are:
+
+    .. math::
+
+       I_{pH,aa}=
+       \begin{cases}
+         \exp{-3 (\frac{pH - pH_{UL,aa}}{pH_{UL,aa} - pH_{LL,aa}})^2} & \text{for } pH \le pH_{UL,aa}\\
+         1 & \text{for } pH > pH_{UL,aa}
+       \end{cases}
+
+       I_{pH,ac}=
+       \begin{cases}
+         \exp{-3 (\frac{pH - pH_{UL,ac}}{pH_{UL,ac} - pH_{LL,ac}})^2} & \text{for } pH \le pH_{UL,ac}\\
+         1 & \text{for } pH > pH_{UL,ac}
+       \end{cases}
+
+       I_{pH,aa}=
+       \begin{cases}
+         \exp{-3 (\frac{pH - pH_{UL,h2}}{pH_{UL,h2} - pH_{LL,h2}})^2} & \text{for } pH \le pH_{UL,h2}\\
+         1 & \text{for } pH > pH_{UL,h2}
+       \end{cases}
+
```

Comment:
I believe we need a note to explain Z_h2s as a parameter to represent h2s concentration

Reply:
Just to clarify, does this modified form of ADM1 account for H2S in gas produced by the anaerobic digestor?

Reply:
No, we assume the H2S concentration is 0

Reply:
But the fact that we even include its concentration indicates that it doesn't necessarily have to be 0 although the default might be 0. Am I missing something?

Reply:
Sounds good to me, especially if S_h2s is accounted for directly within the PCM additions that we are considering.

Reply:
Off the top of my head, I don't recall if any PCMs address tracking S_h2s, but I assume we could find it if we looked for it.